spark/v3.3/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DropBranchExec.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.sql.execution.datasources.v2
+
+import org.apache.iceberg.spark.source.SparkTable
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.expressions.Attribute
+import org.apache.spark.sql.connector.catalog.Identifier
+import org.apache.spark.sql.connector.catalog.TableCatalog
+
+case class DropBranchExec(catalog: TableCatalog,
+                          ident: Identifier,
+                          branch: String) extends LeafV2CommandExec {
```

Comment:
[style] can we follow the indentation we do for existing operators. i.e new line should start with 4 spaces. for ex: CallExec
https://github.com/apache/iceberg/blob/531a81581aed152bdbcfa8801593f26918dc1d02/spark/v3.1/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/CallExec.scala#L27-L30

Here is scala style , i think spark follows https://github.com/databricks/scala-style-guide

Reply:
Good catch yeah I created an issue last week https://github.com/apache/iceberg/issues/6736 for scala style. The databricks style guide looks a well established one. Right now there's no tooling to enforce it so the tracking issue was created.